internal/pkg/aws/iface/iface.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package iface wraps AWS SDK API functionalities.
+package iface
```

Comment:
I feel like these interfaces should be defined in the packages where they're needed. This would encourage dependency wrapping/isolation as well.

Reply:
Agreed :) I don't think we need to make changes to the `aws` pkg.

If we're looking into grouping all the private interfaces in the `cli` pkg, I think it would be nice to put them all in `cli.go` for example. Right now it can be difficult to find the existing interfaces in a given package.